src/config/sample-data.js

```diff
@@ -0,0 +1,336 @@
+  }, {
+    day: moment().subtract(25, 'days').format('YYYY-MM-DD'),
+    count: 17
+  }, {
+    day: moment().subtract(24, 'days').format('YYYY-MM-DD'),
+    count: 13
+  }, {
+    day: moment().subtract(23, 'days').format('YYYY-MM-DD'),
+    count: 15
+  }, {
+    day: moment().subtract(22, 'days').format('YYYY-MM-DD'),
+    count: 28
+  }, {
+    day: moment().subtract(21, 'days').format('YYYY-MM-DD'),
+    count: 22
+  }, {
+    day: moment().subtract(20, 'days').format('YYYY-MM-DD'),
+    count: 37
+  }, {
+    day: moment().subtract(19, 'days').format('YYYY-MM-DD'),
+    count: 7
+  }, {
+    day: moment().subtract(18, 'days').format('YYYY-MM-DD'),
+    count: 8
+  }, {
+    day: moment().subtract(17, 'days').format('YYYY-MM-DD'),
+    count: 11
+  }, {
+    day: moment().subtract(16, 'days').format('YYYY-MM-DD'),
+    count: 12
+  }, {
+    day: moment().subtract(15, 'days').format('YYYY-MM-DD'),
+    count: 10
+  }, {
+    day: moment().subtract(14, 'days').format('YYYY-MM-DD'),
+    count: 32
+  }, {
+    day: moment().subtract(13, 'days').format('YYYY-MM-DD'),
+    count: 23
+  }, {
+    day: moment().subtract(12, 'days').format('YYYY-MM-DD'),
+    count: 20
+  }, {
+    day: moment().subtract(11, 'days').format('YYYY-MM-DD'),
+    count: 18
+  }, {
+    day: moment().subtract(10, 'days').format('YYYY-MM-DD'),
+    count: 15
+  }, {
+    day: moment().subtract(9, 'days').format('YYYY-MM-DD'),
+    count: 7
+  }, {
+    day: moment().subtract(8, 'days').format('YYYY-MM-DD'),
+    count: 6
+  }, {
+    day: moment().subtract(7, 'days').format('YYYY-MM-DD'),
+    count: 10
+  }, {
+    day: moment().subtract(6, 'days').format('YYYY-MM-DD'),
+    count: 12
+  }, {
+    day: moment().subtract(5, 'days').format('YYYY-MM-DD'),
+    count: 14
+  }, {
+    day: moment().subtract(4, 'days').format('YYYY-MM-DD'),
+    count: 24
+  }, {
+    day: moment().subtract(3, 'days').format('YYYY-MM-DD'),
+    count: 20
+  }, {
+    day: moment().subtract(2, 'days').format('YYYY-MM-DD'),
+    count: 27
+  }, {
+    day: moment().subtract(1, 'days').format('YYYY-MM-DD'),
+    count: 36
+  }, {
+    day: moment().format('YYYY-MM-DD'),
+    count: 10
+  }],
+  stats: {
+    uniqueAppUsers: {
+      count: 50,
+      previousPeriodCount: 35
+    },
+    totalAppUsers: {
+      count: 60,
+      previousPeriodCount: 40
+    },
+    appSessions: {
+      count: 4913,
+      previousPeriodCount: 5000
+    },
+    studioSessions: {
+      count: 430,
+      previousPeriodCount: 500
+    },
+    studioUsers: {
+      count: 2,
+      previousPeriodCount: 1
+    },
+    newStudioUsers: {
+      count: 1,
+      previousPeriodCount: 0
+    },
+    appsCreated: {
+      count: 2,
+      previousPeriodCount: 1
+    },
+    appsEdited: {
+      count: 1,
+      previousPeriodCount: 1
+    },
+    appsPublished: {
+      count: 1,
+      previousPeriodCount: 0
+    }
+  },
+  apps: [{
+    id: 1,
+    name: 'First app',
+    createdAt: '2021-09-07T11:27:08.321Z',
```

Comment:
These are dates as well. Can we make them relative to the user's current timestamp?